grunt_tasks/plugin.js

```diff
@@ -150,26 +191,86 @@ module.exports = function (grunt) {
                     }
                 }
             });
+
+            // If the plugin config has no webpack section, no defaultLoaders
+            // property in the webpack section, or the defaultLoaders is
+            // explicitly set to anything besides false, then augment the
+            // webpack loader configurations with the plugin source directory.
+            if (!config.webpack || config.webpack.defaultLoaders === undefined || config.webpack.defaultLoaders !== false) {
+                console.log('using!');
+                var numLoaders = grunt.config.get('webpack.options.module.loaders').length;
+                for (var i = 0; i < numLoaders; i++) {
+                    var selector = 'webpack.options.module.loaders.' + i + '.include';
+                    var loaders = grunt.config.get(selector);
+                    grunt.config.set(selector, loaders.concat([path.resolve(dir)]));
+                }
+            }
+
+            var newConfig = webpackHelper(grunt.config.get('webpack.options'), helperConfig);
+            grunt.config.set('webpack.options', newConfig);
         }
 
+        grunt.registerTask('npm-install', 'Install plugin NPM dependencies', function (plugin) {
+            var done = this.async();
+
+            var target = path.resolve('node_modules_' + plugin);
+
+            if (fs.existsSync(target + '/node_modules')) {
+                grunt.log.writeln('NPM dependencies for plugin ' + plugin + ' already installed.');
+                done();
+            }
+
+            var npm = require('npm');
+            var npmConf = {
+                prefix: target,
+                g: true
```

Comment:
`npm` doesn't have an option that's fully like autoconf's `--prefix=<path>` to change the place where things get installed. For some reason, the `--prefix` option has to be combined with `-g` (which is normally used to do a global install) or else it doesn't work.

My knowledge on this particular quirk comes directly from Stack Overflow, so if you think it's necessary I can try to do some further research on it.